scripts/make/qemu.mk

```diff
@@ -59,7 +67,9 @@ qemu_args-$(GRAPHIC) += \
   -device virtio-gpu-$(vdev-suffix) -vga none \
   -serial mon:stdio
 
-ifeq ($(GRAPHIC), n)
+ifeq ($(CONSOLE), y)
+  qemu_args-y += -display none
+else ifeq ($(GRAPHIC), n)
```

Comment:
is it `if-else if` rather than `if-if`?

Reply:
if using `CONSOLE=y` and `GRAPHIC=n` at the same time, will it be right?

Reply:
If -nographic is used, qemu will implicitly open a new chardev, which will compete with virtconsole, and the qemu compilation option will not pass, with the error: cannot use stdio by multiple character devices. display-none only does not display the graphical interface, the graphical interface still exists, will not open a new chardev, because the effect of the two is similar in most cases, so open one.